src/Tizen.NUI/src/internal/Interop/Interop.NativeImageSource.cs

```diff
@@ -27,6 +27,9 @@ internal static partial class NativeImageSource
 
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_NativeImageSource_ReleaseBuffer")]
             public static extern bool ReleaseBuffer(IntPtr jarg1);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_NativeImageSource_GenerateUrl")]
+            public static extern string GenerateUrl(IntPtr handle);
         }
     }
 }
```

Comment:
could you add new line here? 
(https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file)

Reply:
Thank you I fixed it